web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -246,16 +256,22 @@
     playerInitialized = false;
   };
 
+  const resetAndPlay = () => {
+    handlePromiseError(handleAction('resetAndPlay', 'reset'));
+    handlePromiseError(handleAction('resetAndPlay', 'play'));
```

Comment:
You have changed the first parameter here. Why?

Reply:
The first parameter is only for logging, previously it was `initPlayer[AssetViewOpen]` since that was the name of the method it was contained in.